src/image_bios.cpp

```diff
@@ -18,9 +17,14 @@
 
 static const fs::path aspeedSMC = "/sys/bus/platform/drivers/aspeed-smc";
 static constexpr auto spiDriver = "1e631000.spi";
+static const char* mtdDevice = "/dev/mtd/bios";
+static const size_t mtdBlockSize = 0xffff;
 static constexpr auto gpioNamePCHPower = "PWRGD_DSW_PWROK";
 static constexpr auto gpioNameBIOSSel = "MIO_BIOS_SEL";
 static constexpr auto gpioOwner = "fwupdate";
+static constexpr size_t nvramOffset = 0x01000000;
+static constexpr size_t nvramSize = 0x00080000;
```

Comment:
I believe we need to make this configurable in case of future changes in our UEFI size / layout or just new machines with different layouts. This must be a machine level configuration in OpenBMC, so this project needs to support that. The `mtdDevice` and `mtdBlockSize` may also need to be externally configurable.

Reply:
I agree that such configuration should not be hardcoded.
But on the other hand, for this YADRO-specific project, we already have about 30 configurable parameters that never change. And I believe that they won't, because it is easier to make change in one line of source code than create bbappend file.
I would prefer to make these parameters configurable only if we really need it, and not if "maybe it will be needed sometime".
Are you still insisting on that?